Wikibase/Tests/Integration/LicenseRequired.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+// phpcs:property Wikibase\Sniffs\Commenting\ClassLevelDocumentationSniff::$license = MIT
+// phpcs:disable Generic.Files.OneObjectStructurePerFile
```

Comment:
this is disabled only to allow multiple test "cases" in a single file, right?

Reply:
I added this line to reduce the amount of messages in LicenseRequired.php.expected that are not relevant for this test.